src/scss/06_components/paragraphs/_contact-card.scss

```diff
@@ -0,0 +1,78 @@
+.contact-card {
+  display: flex;
+}
+
+.contact-card--with-image {
+  @include breakpoint($breakpoint-s) {
+    flex-direction: column;
+  }
+}
+
+.contact-card--information-container {
+  @include breakpoint($breakpoint-s) {
+    max-width: 200px;
+  }
+}
+
+.contact-card--person-social-media {
+  display: flex;
+
+  .hel-icon {
+    height: 26px;
+    margin-right: $spacing-half;
+    width: 26px;
+  }
+}
+
+.contact-card--image-container {
+  margin-bottom: $spacing;
+  margin-right: $spacing;
+
+  @include breakpoint($breakpoint-s) {
+    margin-right: 0;
+  }
+
+  img {
+    height: 100px;
```

Comment:
If these numbers come from design, I'd recommend putting them to variables that explain their meaning and using that variable. Or adding a comment.

Reply:
Yep, comes from the design as well as the 200px. It'll change these to variables